src/printer/index.js

```diff
@@ -333,60 +333,41 @@ function printWithItem(path, print) {
 }
 
 function printIf(path, print) {
-  function printOrElse(path) {
-    const n = path.getValue();
-
-    if (n.orelse && n.orelse.length > 0) {
-      if (n.orelse[0].ast_type === "If") {
-        return concat(path.map(printElseBody, "orelse"));
-      }
-      return concat([
-        hardline,
-        "else:",
-        concat(path.map(printElseBody, "orelse"))
-      ]);
-    }
-  }
-
-  function printElseBody(path) {
-    const n = path.getValue();
-
-    const parts = [];
-
-    if (n.ast_type === "If") {
-      parts.push(
-        concat([
-          hardline,
-          "elif ",
-          path.call(print, "test"),
-          ":",
-          indent(concat([hardline, printBody(path, print)]))
-        ])
-      );
-    } else {
-      parts.push(indent(concat([hardline, path.call(print)])));
-    }
-
-    const orElse = printOrElse(path);
+  const parent = path.getParentNode();
+  const n = path.getValue();
 
-    if (orElse) {
-      parts.push(orElse);
-    }
+  let type = "if";
 
-    return concat(parts);
+  if (
+    parent.ast_type === "If" &&
+    parent.orelse.length > 0 &&
+    parent.orelse[0].ast_type === "If" &&
+    parent.orelse[0] === n
+  ) {
+    type = "elif";
   }
 
   const parts = [
-    "if ",
+    type,
+    " ",
     path.call(print, "test"),
     ":",
     indent(concat([hardline, printBody(path, print)]))
   ];
 
-  const orElse = printOrElse(path);
+  if (n.orelse.length) {
+    parts.push(hardline);
 
-  if (orElse) {
-    parts.push(orElse);
+    if (n.orelse[0].ast_type === "If") {
```

Comment:
Similar to above, I think this need to also check that `n.orelse.length === 1`. In fact we might want to pull this out to a `is_elif` function that operates on the parent `If` AST node (where "parent" here is `n`, but earlier in the function is `parent`). Or maybe a `get_elif_node` which takes in the parent and either returns the single `If` node in the `orelse` or `null` if it's not an elif node. That would allow us to check for `get_elif_node(parent) === n` above and `get_elif_node(n) !== null` here.

Also I'd like to see a comment here explaining why we do it this way. I had to think for a bit to understand why we can't have this `n.orelse[0].ast_type === "If"` logic only once. Maybe something like

> If this is going to be an `elif` we can't print `else:` and indent, but rather let the child print `elif` and indent itself.